### DISCUSSION TITLE
Normalization method

### DISCUSSION THREAD

**Discussion by sujan1746** [ROBelgium/MSNoise#306] [Q&A]
Hi, I have used msnoise very well and thank you so much for your help.

I want to ask you the reason of using 'clipping waveform' for normalization.

MSNoise usually clips n*RMS which is one method mentioned in Bensen paper(2007) (1-bit is also able to be used in MSNoise but not recommended when I asked before).

In Bensen paper, "running absolute mean normalization" is recommended and "water level normalization" is smilar to "clipping" but just down-weighted. And I heard the clipping method has more possibility to ruin waveform.

I just curious the reason that you made the program with using 'clipping' instead of using 'running absolute mean normalization' or 'water level normalization' and its advantages.

**Comment by ThomasLecocq** [comment]
it was a design choice at the origin of the code (2010...)  - there is a branch here that implements the absolute running mean. Note also that each window is clipped individually, not the full day and then windowed. The clipping can indeed induce ringing effects when computing a FFT transform, but most of the times that doesn't really damage the signal. 1-bit is a different beast. 

I think @asyates could elaborate a bit here too, since he did lots of tests on that matter

**Comment by asyates** [comment]
Hello hello,

In fact, i haven't tested the running absolute mean normalization (or water level normalization), i've mostly worked with 1-bit and RMS clipping.

On this, for 1-bit i've certainly seen how it can be problematic if done prior to spectral whitening (can basically eliminate certain frequencies from the signal if they are small amplitude).

Also seen in some tests that the SNR seems to be slightly higher when applying RMS vs 1-bit, though i also recall a New Zealand study where they found the opposite. I think the reasoning was that 1-bit better dealt with a dataset that included lots of local earthquakes, though not sure this was mentioned in any publication that followed. But basically, the best choice may be context specific.

Would be interested to see examples where there is a clear preference for one method over another. As Tom said, most of the time it seems that we get acceptable results using RMS clipping so this is what i tend to use.
